test-suite/soak-tests/README.md

```diff
@@ -6,9 +6,12 @@ This is an internal crate for subjecting Exonum networks to [soak tests].
 
 The crate exposes the following binaries:
 
-- [`toggle`](src/toggle.rs). Tests repeatedly switching a service on and off.
+- [`toggle`](src/bin/toggle.rs). Tests repeatedly switching a service on and off.
   The service generates transactions in the `after_commit` hook.
 
+- [`send_txs`](src/bin/send_txs.rs). Tests sending transactions to a service
```

Comment:
I wonder can we use this test to approximately estimate TPS and then print it in summary?

Reply:
Well yeah, but this is blocked by the transaction sending being hacky (which I plan to fix separately after transition to tokio 0.2).